.github/workflows/ci.yml

```diff
@@ -128,3 +128,23 @@ jobs:
           cmake .. -G "MinGW Makefiles" -DBUILD_TESTING=ON -DBUILD_TESTING_INTEGRATION=ON -DC_WARNINGS_AS_ERRORS=ON -DCPP_WARNINGS_AS_ERRORS=ON
           cmake --build .
           ctest --verbose -R "allocator|value|example_basic|integration_basic"
+
+  build_and_test_linux_wasm:
+    strategy:
+     matrix:
+        platform: [ubuntu-18.04]
+    runs-on: ${{ matrix.platform }}
+    env:
+      MG_VERSION: "1.4.0"
+    steps:
+      - name: Set-up repository
+        uses: actions/checkout@v2
+      - name: Install environment
+        run: |
+          sudo apt install -y clang-8 clang++-8
+
```

Comment:
Either separate every action with a newline, or none, just for the sake of consistency

Reply:
Good comment, the code changed slightly but I followed your advice ;)